packages/mui-material/src/styles/createTheme.test.js

```diff
@@ -5,10 +5,43 @@ import Button from '@mui/material/Button';
 import Box from '@mui/material/Box';
 import { ThemeProvider, createTheme, styled } from '@mui/material/styles';
 import { deepOrange, green } from '@mui/material/colors';
+import createPalette from './createPalette';
+
+const lightPalette = createPalette({ mode: 'light' });
+const darkPalette = createPalette({ mode: 'dark' });
 
 describe('createTheme', () => {
   const { render } = createRenderer();
 
+  it('should not have custom properties', () => {
+    const theme = createTheme({ cssVariables: false });
+    expect(theme.cssVariables).to.equal(false);
+    expect('vars' in theme).to.equal(false);
+  });
+
+  it('color schemes dark: true', () => {
+    const theme = createTheme({ cssVariables: false, colorSchemes: { dark: true } });
+    const { light, dark } = theme.colorSchemes || {};
```

Comment:
```suggestion
    const { light, dark } = theme.colorSchemes;
```

Shouldn't this just work?

Reply:
Yes, I forgot to remove it.